documentation/source/building.rst

```diff
@@ -34,10 +34,10 @@ Make Variables
       Selects which simulator Makefile to use.  Attempts to include a simulator specific makefile from :file:`cocotb/makefiles/makefile.$(SIM)`
 
     ``VERILOG_SOURCES``
-      A list of the Verilog source files to include.
+      A list of the Verilog source files to include. Paths can be absolute or relative; if relative, they are interpreted as relative to the Makefile's location.
```

Comment:
Nit: we typically have one sentence per line.

Reply:
Will fix and rebase!